utils/merkletrie/iter_test.go

```diff
@@ -0,0 +1,462 @@
+package merkletrie_test
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"srcd.works/go-git.v4/utils/merkletrie"
+	"srcd.works/go-git.v4/utils/merkletrie/internal/fsnoder"
+	"srcd.works/go-git.v4/utils/merkletrie/noder"
+)
+
+func Test(t *testing.T) { TestingT(t) }
```

Comment:
The `Test` hook should go ideally in the "main" test file of this package. Is `iter_test.go` the "main" one?